### ISSUE TITLE
Support for Visual Studio 2022

### ISSUE THREAD

**Issue by ldrick** [Acumatica/Acuminator#410]
Hi, some of our team like to switch to Visual Studio 2022 already (me included) and Acuminator is one of the extensions with high usage. Is there already a release planed for Acuminator 2022?

Thank you very much

**Comment by SENya1990** [comment]
@ldrick Hi. In short, we have plans to add this support but currently it is hard to predict when it will be added. It is in our list of high priority Acuminator tasks.

 VS 2022 contains huge breaking changes in the VS SDK. Basically it requires to rewrite a huge piece of the VS extension to support VS 2022. Also the extension for VS 2022 has to be a separate package. You can see more details here:
https://docs.microsoft.com/en-us/visualstudio/extensibility/migration/update-visual-studio-extension?view=vs-2022

All this means that a lot of work has to be done to add the support for VS 2022. 
There is no dedicated team in Acumatica to support the Acuminator. It is maintained by the platform team with the other features and support requests. Therefore, unfortunately, there is some lack of resources, we can't switch to Acuminator and start adding this support immediately.  We will discuss the addition of this support during the planning for the future quarters but currently I have no dates for you. Maybe @dropsonic knows better.

**Comment by ldrick** [comment]
Thanks @SENya1990 no pressure from our side. We can still use 2019 or both in parallel. And now we've got a slight idea on how much effort it takes from your side. Again thank you for the hard work

**Comment by SENya1990** [comment]
Here is a beta for Acuminator extension that works with VS 2022. **It works only for VS 2022, for earlier versions of VS use thhe released Acuminator extension from [VS marketplace](https://marketplace.visualstudio.com/items?itemName=Acumatica.Acuminator)**
Huge thanks to @shawnburt  for it. 
[Acuminator beta VS2022.zip](https://github.com/Acumatica/Acuminator/files/7629198/Acuminator.beta.VS2022.zip)

The addition of support for VS 2022 currently is still in progress

**Comment by SENya1990** [comment]
There is a new PR with Acuminator support for VS 2017, VS 2019 and VS 2022 in one package
